src/styles/blocks/menu.scss

```
@@ -0,0 +1,67 @@
+.menu {
+  background-image: url(../images/header-full-background.svg);
+  background-size: 400%;
+
+  :nth-child(n) {
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    color: $main-black;
+    letter-spacing: 2px;
+  }
+
+  &__top {
+    margin-bottom: 55px;
+    @include actions;
+  }
+
+  &__list {
+    margin: 0 0 40px;
+  }
+
+  &__item {
+    margin-bottom: 24px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__content {
+    box-sizing: border-box;
+    height: 100vh;
+    padding: $headerTopPadding 0;
+  }
+
+  &__link {
+    display: inline-block;
+    padding-bottom: 8px;
+    border-bottom: 1px solid transparent;
+
+    font-size: 22px;
+    line-height: 22px;
+    transition: border-color 0.3s;
+
+    &:hover {
+      border-bottom: $main-black solid;
```

```suggestion
      text-decoration: underline;
```
bc border affects other elements